package/ngx-devel-kit/ngx-devel-kit.mk

```diff
@@ -0,0 +1,15 @@
+################################################################################
+#
+# ngx_devel_kit
+#
+################################################################################
+
+NGX_DEVEL_KIT_VERSION = 0.3.2
+NGX_DEVEL_KIT_SITE = $(call github,vision5,ngx_devel_kit,$(NGX_DEVEL_KIT_VERSION))
+NGX_DEVEL_KIT_LICENSE = BSD-2-Clause
```

Comment:
this is not the correct license. The correct license is: https://github.com/vision5/ngx_devel_kit
Can you please check the licences also for the other packages you added?

Reply:
Done